app/code/core/Mage/Directory/Model/Resource/Currency.php

```diff
@@ -183,16 +183,22 @@ public function saveRates($rates)
      */
     public function getConfigCurrencies($model, $path)
     {
-        $adapter = $this->_getReadAdapter();
-        $bind    = array(':config_path' => $path);
-        $select  = $adapter->select()
-                ->from($this->getTable('core/config_data'))
-                ->where('path = :config_path');
-        $result  = array();
-        $rowSet  = $adapter->fetchAll($select, $bind);
-        foreach ($rowSet as $row) {
-            $result = array_merge($result, explode(',', $row['value']));
+        $result  = [];
+        $config = Mage::app()->getConfig();
```

Comment:
this looks like a change from https://github.com/OpenMage/magento-lts/pull/1252/files, it shouldn't be here, right?

Reply:
Thank you! I mischeckouted to another branch :-)